app/handlers/objectTracking.py

```diff
@@ -17,5 +27,56 @@ class ObjectTrackingHandler(tornado.web.RequestHandler):
 
     @apiError error_message The error message to display.
     """
+
     def post(self):
+        self.objectTrack(self.request.body_arguments["identifier"])
+        
+        message = "Hello,\n\tWe have finished processing your video and identifying all objects.\nThank you for your patience,\nThe Santos Team"
+        subject = "Your video has finished processing."
+
+        EmailHelper.send_email(self.request.body_arguments["email"], subject, message)
```

Comment:
It would be cool if we had a single file that contained helper methods for sending each email for the analysis.